custom/icds_reports/reports/maternal_child.py

```diff
@@ -36,16 +36,16 @@ def get_data_for_child_health_monthly(date, filters):
             {'age_tranche__lt': 72},
             wasting_severe_column(icds_feature_flag)
         )
-        stunting_moderate = exclude_records_by_age_for_column(
-            age_filters,
+        stunting_moderate = include_records_by_age_for_column(
+            {'age_tranche__lt': 72},
             stunting_moderate_column(icds_feature_flag)
         )
-        stunting_severe = exclude_records_by_age_for_column(
-            age_filters,
+        stunting_severe = include_records_by_age_for_column(
+            {'age_tranche__lt': 72},
             stunting_severe_column(icds_feature_flag)
         )
         nutrition_status_weighed = exclude_records_by_age_for_column(
-            age_filters,
+             age_filters,
```

Comment:
This column `nutrition_status_weighed` and below `height_measured_in_month` are the denominator of underweight and stunting. Therefore this constraint update also needs to applied on these.

Reply:
Updated these properties accordingly and updated relevant test cases as well, please review. Thanks!